Resources/Prototypes/_NF/Entities/Objects/Storage/pouches.yml

```
@@ -0,0 +1,1323 @@
+      map: [ "pouchFill3" ]
+      visible: false
+    - state: fill-util2-1
+      map: [ "pouchFill2" ]
+      visible: false
+    - state: fill-util1-1
+      map: [ "pouchFill1" ]
+      visible: false
+    - state: decal-line-01
+      map: [ "decal1" ]
+      color: "#b33831"
+  - type: Item
+    inhandVisuals:
+      left:
+      - state: inhand-left
+        color: "#4960ed"
+      right:
+      - state: inhand-right
+        color: "#4960ed"
+  - type: Clothing
+    clothingVisuals:
+      belt:
+      - state: equipped-BELT
+        color: "#4960ed"
+      pocket:
+      - state: equipped-BELT
+        color: "#4960ed"
+
+## WILDCARDS
+- type: entity
+  id: NFPouchMercenary
+  parent: NFPouchUtility
+  name: mercenary's pouch
+  components:
+  - type: Sprite
+    layers:
+    - state: base
+      color: "#3e3e48"
+    - state: closed
+      map: ["closeLayer"]
+      color: "#7b7b3f"
+    - state: open
+      color: "#7b7b3f"
+      map: ["openLayer"]
+      visible: false
+    - state: inside
+      map: ["insideLayer"]
+      visible: false
+    - state: fill-util4-1
+      map: [ "pouchFill4" ]
+      visible: false
+    - state: fill-util3-1
+      map: [ "pouchFill3" ]
+      visible: false
+    - state: fill-util2-1
+      map: [ "pouchFill2" ]
+      visible: false
+    - state: fill-util1-1
+      map: [ "pouchFill1" ]
+      visible: false
+  - type: Item
+    inhandVisuals:
+      left:
+      - state: inhand-left
+        color: "#7b7b3f"
+      right:
+      - state: inhand-right
+        color: "#7b7b3f"
+  - type: Clothing
+    clothingVisuals:
+      belt:
+      - state: equipped-BELT
+        color: "#7b7b3f"
+      pocket:
+      - state: equipped-BELT
+        color: "#7b7b3f"
+
+- type: entity
+  id: NFPouchPrivateSecurity
+  parent: NFPouchUtility
+  name: private security pouch
+  components:
+  - type: Sprite
+    layers:
+    - state: base
+      color: "#935c3f"
+    - state: closed
+      map: ["closeLayer"]
+      color: "#2286bd"
+    - state: open
+      color: "#2286bd"
+      map: ["openLayer"]
+      visible: false
+    - state: inside
+      map: ["insideLayer"]
+      visible: false
+    - state: fill-util4-1
+      map: [ "pouchFill4" ]
+      visible: false
+    - state: fill-util3-1
+      map: [ "pouchFill3" ]
+      visible: false
+    - state: fill-util2-1
+      map: [ "pouchFill2" ]
+      visible: false
+    - state: fill-util1-1
+      map: [ "pouchFill1" ]
+      visible: false
+    - state: decal-line-01
+      map: [ "decal1" ]
+      color: "#9d1c1c"
+  - type: Item
+    inhandVisuals:
+      left:
+      - state: inhand-left
+        color: "#935c3f"
+      right:
+      - state: inhand-right
+        color: "#935c3f"
+  - type: Clothing
+    clothingVisuals:
+      belt:
+      - state: equipped-BELT
+        color: "#935c3f"
+      pocket:
+      - state: equipped-BELT
+        color: "#935c3f"
+
+## NFSD
+- type: entity
+  id: NFPouchNfsd
+  parent: NFPouchUtility
+  name: nfsd pouch
```

```suggestion
  name: NFSD pouch
```
caps in description for consistency